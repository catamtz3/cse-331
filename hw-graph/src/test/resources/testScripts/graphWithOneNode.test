# Create a graph
CreateGraph singleNode

#Add node to the graph
AddNode singleNode a

# Get the nodes of the graph
ListNodes singleNode