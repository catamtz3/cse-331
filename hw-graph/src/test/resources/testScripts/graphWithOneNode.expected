# Create a graph
created graph singleNode

#Add node to the graph
added node a to singleNode

# Get the nodes of the graph
singleNode contains: a